src/test/suite/concatTextDocument.test.ts

```diff
@@ -25,7 +35,7 @@ suite('concatTextDocument', () => {
             [

```

Comment:
Sorry rest of this file is formatting again. I added some extra tests to verify the score.